docs/en/ingest-management/fleet/set-inactivity-timeout.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[set-inactivity-timeout]]
+= Set inactivity timeout
+
+[[why-set-inactivity-timeout]]
+== Why set an inactivity timeout?

```

Comment:
```suggestion
```
Not sure if this ID is referenced somewhere else, but I don't think we need this heading here. It just sort of repeats the topic title as a question. Also the "why" is a bit buried in the paragraph. I would just delete this.